src/theme.css

```diff
@@ -21,7 +21,7 @@
   --rd-spc-xxxs: 0.0625rem;
   --rd-spc-xxs: 0.25rem;
   --rd-spc-xs: 0.5rem;
-  --rd-spc-s: 1rem;
+  --rd-spc-s: 0.75rem;
```

Comment:
this is a site-wide spacing. You sure you want to commit that change?

Reply:
well, everything looks alright to me.

Reply:
Yeah, I looked where it was used, and it's safe :)